acquire/tools/decrypter.py

```diff
@@ -424,6 +421,18 @@ def main():
                     break
 
 
+def find_enc_files(files: list[Path]):
+    output_files = []
```

Comment:
These are not output files, it searches for encrypted files. Maybe they are used later on to construct output files, but that is not in the context of this function